main.tf

```
@@ -15,11 +15,11 @@ locals {
         for k8s_ip_ranges in var.ip_ranges.private_k8s : [
           {
             cidr = k8s_ip_ranges.pods,
-            name = format("%s-k8spods-%s", coalesce(k8s_ip_ranges.name, "private"), var.name_suffix)
+            name = format("%s-%s", coalesce(k8s_ip_ranges.pods_name, "private"), var.name_suffix)
```

```suggestion
            name = format("%s-%s", coalesce(k8s_ip_ranges.pods_name, "private-k8spods"), var.name_suffix)
```
This will allow most existing IACs to simply upgrade without having to specify any new name value